openff/qcsubmit/factories.py

```diff
@@ -96,13 +104,13 @@ def clear_workflow(self) -> None:
         """
         Reset the workflow to by empty.
         """
-        self.workflow = {}
+        self.workflow = []
 
     def add_workflow_component(
```

Comment:
Would it maybe make sense to change this to `add_workflow_components` a maybe allow it to accept an args list of components rather than just a list? i.e. `add_workflow_components(*components: Component)` that way you could write

```
factory.add_workflow_components(component_1, component_2, ...)
```

without needing to wrap them in a list?

Reply:
Great idea adding this in.